index.html

```diff
@@ -63,140 +68,141 @@ <h2 class="visually-hidden">NFT CARDS</h2>
                 <li class="nft-card">
                     <a href="#">

```

Comment:
In Figma, this element has a background color (white with transparency). Please note that you need to add background colors to all elements that have them in Figma.